test/ref_ops_nonstd_shape_test.cpp

```diff
@@ -55,4 +55,62 @@ TEST_CASE(argmin_test_nonstd_shape)
     EXPECT(migraphx::verify_range(result_vec, res_gold_vec));
 }
 
+TEST_CASE(squeeze_transpose_test)
+{
+    migraphx::program p;
+    auto* mm = p.get_main_module();
+    std::vector<float> data(4 * 3 * 3);
+    migraphx::shape s1{migraphx::shape::float_type, {4, 1, 3, 1, 3}};
+    auto l0 = mm->add_literal(migraphx::literal{s1, data});
+    auto l0_trans =
+        mm->add_instruction(migraphx::make_op("transpose", {{"permutation", {1, 2, 3, 0, 4}}}), l0);
+    mm->add_instruction(migraphx::make_op("squeeze"), l0_trans);
+    auto p_uncompiled = p;
+    p.compile(migraphx::ref::target{});
+    auto result         = p.eval({}).back();
+    auto* mm_uncompiled = p_uncompiled.get_main_module();
+    // ref implementation always does the auto_contiguous pass
+    migraphx::run_passes(*mm_uncompiled, {migraphx::auto_contiguous{}});
+    auto expected_result = p_uncompiled.eval({}).back();
+    EXPECT(result.get_shape() == expected_result.get_shape());
```

Comment:
You need to check the values are the same: `EXPECT(result == expected_result)`

Reply:
Made the change.